posts/marker-gene-smoothing/index.qmd

```diff
@@ -0,0 +1,714 @@
+---
+title: "Applications for visualization and cell typing using 'smoothed' marker genes" 
+author:
+  - name: Dan McGuire
+    orcid: 0009-0006-0286-9625
+    affiliations:
+      - ref: nstg
+      - ref: dan11mcguire
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 4
+date: "2024-06-07"
+date-modified: last-modified
+categories: [visualization, cell typing]
+image: figures/plot_oneCosMxFOV-1.png
```

Comment:
This plot does not exist in the repo. please `git add`.

Reply:
I changed the thumbnail pic to this one, now tracked in  repo:
![image](https://github.com/Nanostring-Biostats/CosMx-Analysis-Scratch-Space/assets/49620432/acdeed61-1755-4f20-8cdf-e997c0c56013)